exonum-java-binding-core/src/main/java/com/exonum/binding/service/NodeFake.java

```diff
@@ -86,8 +88,9 @@ public NodeFake(MemoryDb database, byte[] publicKey) {
    * @throws NullPointerException if the transaction is null
    */
   @Override
-  public void submitTransaction(Transaction transaction) {
+  public HashCode submitTransaction(RawTransaction transaction) {
     checkNotNull(transaction);
+    return null;
```

Comment:
I'd use a 32-byte HashCode of zeros as the default, `null` doesn't feel right in a `@NonNull` method